Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM        registry.access.redhat.com/ubi8/python-36
+FROM        registry.access.redhat.com/ubi8/python-38
```

Comment:
would python-39 be an option ?

Reply:
It seems to work smoothly with `3.9` too -> bumped the version to `3.9` now.
Set the minimum required version to `3.8`, as this is default on many platforms now.